workflow/controller/operator.go

```diff
@@ -1375,6 +1389,9 @@ func (woc *wfOperationCtx) executeTemplate(nodeName string, orgTmpl wfv1.Templat
 		err = errors.Errorf(errors.CodeBadRequest, "Template '%s' missing specification", processedTmpl.Name)
 		return woc.initializeNode(nodeName, wfv1.NodeTypeSkipped, templateScope, orgTmpl, boundaryID, wfv1.NodeError, err.Error()), err
 	}
+	if apierr.IsForbidden(err) {
```

Comment:
I'm assuming we're assuming that forbidden means - you don't have enough memory or similar?

Reply:
@alexec, yes, we're assuming forbidden means there are not enough resources to schedule a pod.

Reply:
Can we move this inside the `if err != nil {`